addons/netfox/encoder/redundant-history-encoder.gd

```diff
@@ -22,29 +22,38 @@ func set_redundancy(p_redundancy: int):
 
 	redundancy = p_redundancy
 
-func encode(tick: int) -> Array:
+func encode(tick: int, properties: Array[PropertyEntry]) -> Array:
```

Comment:
The argument holding `Array[PropertyEntry]` is called `properties` here, `property_config` in `decode` and in snapshot-history-encoder.gd and `property_entries` in diff-history-encoder.gd. Is there any reason for the discrepancy or can these names be made more consistent?

Reply:
Fixed